synapse/util/constants.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 The Matrix.org Foundation C.I.C.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+HOUR_IN_MS = 60 * 60 * 1000
```

Comment:
Is this used anywhere by the presence code? Why does this need to move?

Reply:
It *was*, but now is no longer. We can back this out until we need to use it elsewhere.